src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/ErrorHandlingTransmissionPolicy.cs

```diff
@@ -59,8 +66,8 @@ private void HandleTransmissionSentEvent(object sender, TransmissionProcessedEve
                             this.backoffLogicManager.ReportBackoffEnabled((int)httpWebResponse.StatusCode);
                             this.Transmitter.Enqueue(e.Transmission);
 
-                            this.backoffLogicManager.ScheduleRestore(
-                               httpWebResponse.RetryAfterHeader,
+                            this.pauseTimer.Delay = this.backoffLogicManager.GetBackOffTimeInterval(httpWebResponse.RetryAfterHeader);
+                            this.pauseTimer.Start(
```

Comment:
then previous is cancelled and after next delay interval capacities will be set back to null - that's exactly what expected